test/Makefile.am

```diff
@@ -1 +1 @@
-SUBDIRS = common performance miscellaneous validation
+SUBDIRS = common validation performance miscellaneous
```

Comment:
For parallel make case you need ".". Might be for the same time it's good to isolate performance tests. I.e.:
SUBDIRS = common validation . performance . miscellaneous
https://www.gnu.org/software/automake/manual/html_node/Subdirectories.html

Reply:
@muvarov hmm, you do not need "." here, as there is nothing executed in the current (test/) directory.

Reply:
@lumag --enable-test-perf enables performance tests. On parallel with 'make check' it's better to not run them with validation tests at the same time.

Reply:
Dot just allows current directory to be built in parallel with the subdirs.

Reply:
than it has to be ok.